webapp/channels/src/components/mfa/confirm.tsx

```diff
@@ -30,57 +30,54 @@ type Props = {
     updateParent: (state: MFAControllerState) => void;
 }
 
-export default class Confirm extends React.PureComponent<Props> {
-    public componentDidMount(): void {
-        document.body.addEventListener('keydown', this.onKeyPress);
-    }
-
-    public componentWillUnmount(): void {
-        document.body.removeEventListener('keydown', this.onKeyPress);
-    }
-
-    submit = (e: KeyboardEvent | React.FormEvent<HTMLFormElement>): void => {
+export default function Confirm({state, updateParent}: Props) {
+    const submit = (e: KeyboardEvent | React.FormEvent<HTMLFormElement>): void => {
         e.preventDefault();
         redirectUserToDefaultTeam();
     };
```

Comment:
Since this doesn't use anything from inside the component, can be defined outside of it. That way we don't need to re-generate it on every render, and it is clearer for dependencies and so on that it is not supposed to change between renders.